docs/reference/lang/java.rst

```diff
@@ -8,15 +8,17 @@ Java
 
     This module is a work in progress.
 
-Java a perhaps the most used programming language in the world.
+Java is perhaps the most used programming language in the world.
```

Comment:
Btw, I'd personally rewrite this sentence at all. Everybody knows that C is the most used language in the world, followed by Python. Java? Bleh. So, making such claims in project docs looks dubious :-D.

Reply:
I'm not too sure about this, the amount of embedded code is large, but enterprise webservices are an endless ocean :)